src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -62,50 +62,87 @@ object SbtSassCompiler extends AutoPlugin {
           (Assets / compileSass / includeFilter).value
             -- (Assets / compileSass / excludeFilter).value
         )
+
         val sassFilesFound  = sourceDir.globRecursive(sassFilesFilter).get.filterNot(_.isDirectory)
-        val logger          = streams.value.log
-        logger.info(s"Sass compiling via sbt-sass-compiler: ${sassFilesFound.length} files")
+        val partialFilesFound = sourceDir.globRecursive((Assets / compileSass / excludeFilter).value).get.filterNot(_.isDirectory)
```

Comment:
I think this will find anything in the assets/ dir that has a leading _ in the filename or is a hidden file

if we just wanted partials it would be intersection of compileSass/includeFilter and compileSass/excludeFilter, rather than the difference (to just get the sass files that aren't partials) - or maybe more clearly, just the intersection of includeFilter and stuff starting with "_*" and not hidden files

Reply:
@oscarduignan Yup that file filters is one thing I still need to fix, thanks!

Reply:
guess we could add a test if there isn't one to check that making a change to a non-sass file in assets doesn't trigger a recompile

Reply:
@oscarduignan good suggestion 👍